src/extension/script.ts

```diff
@@ -0,0 +1,147 @@
+import { fetchStream, streamBody } from "./fetchStream";
+
+import { getCurrentTab } from "./utils";
+
+async function cherryPickCommits(path, commitBranch, targetBranch) {
+  const table = document.querySelector(".table");
```

Comment:
cherryPickCommits should also receive the commits list.

Reply:
This will be changed in the next PR